tests/examples/factory/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pathmind==0.3
+ruamel.yaml
```

Comment:
Try this. the installed ruamel.yaml is an old version so let's install a new one.

```suggestion
ruamel.yaml>=0.17
```